lib/timeline/Core.js

```diff
@@ -393,13 +393,7 @@ Core.prototype._create = function (container) {
  */
 Core.prototype.setOptions = function (options) {
   if (options) {
-    // copy the known options
-    var fields = [
-      'width', 'height', 'minHeight', 'maxHeight', 'autoResize',
-      'start', 'end', 'clickToUse', 'dataAttributes', 'hiddenDates',
-      'locale', 'locales', 'moment', 'rtl', 'zoomKey', 'horizontalScroll', 'verticalScroll'
-    ];
-    util.selectiveExtend(fields, this.options, options); 
```

Comment:
Isn't it possible that the defective `selectiveExtend` (fixed by https://github.com/visjs/vis-util/pull/15 ) here caused all the subsequent problems? Because I just can't find another bug.

Reply:
I've updated the `vis-util` version but there still seem to be many problems (same as before this PR)

Reply:
Souldn't be the `vis.util` update be part of this PR (in the package.json)? I can't see it...